### DISCUSSION TITLE
How to revert my pendrive from WS- Live to normal pendrive ?

### DISCUSSION THREAD

**Discussion by hardaatbaath** [FedoraQt/MediaWriter#593] [General]
I used my pen drive for Fedora Media Writer, but now I want to use it as a normal pen drive, how to do so?
The Fedora Media Writer is not showing any options to revert it back like it is supposed to.
I am using MacOS Ventura 13.2.1

**Comment by grulja** [comment]
I opened a bug here: https://github.com/FedoraQt/MediaWriter/issues/594.
